source/operations/install-deploy-manage/multi-site-replication.rst

```diff
@@ -108,6 +107,13 @@ For :ref:`SSE-S3 <minio-encryption-sse-s3>` or :ref:`SSE-KMS <minio-encryption-s
 
 You can achieve this with a central KES server or multiple KES servers (say one per site) connected via a central supported :ref:`key vault server <minio-sse>`.
 
+Load Balancers Installed on Each Multi-Node Site
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When replication to multi-node sites, use the URL or IP address of the site's load balancer, reverse proxy, or similar network control plane component which automatically routes requests to nodes in the deployment.
+
+For such environments, pointing to a specific node within the deployment creates a single point of failure risk for replication.
```

Comment:
"For such environments" I *think* is referencing the load balancer env which is already optimal.

It's enough to state:

"Using a single node for configuring site replication creates a single point of failure, where that node being offline results in replication failure."

Reply:
Same throughout. You could pop this into `common-replication.rst` for single sourcing.

Reply:
Good rewording. Thanks.
Also, moved to common-replication as suggested.